### PR TITLE
Fix schema

### PR DESCRIPTION
**Pull request by ChrisRuff**
Fixed the change of models which will be overwritten by codegen

### CODE REVIEW

File: lib/gateway.dart [hunk 1]
```diff
@@ -344,15 +344,21 @@ class Gateway {
...
       }
       else {
         organization?.parkingLots?.add(lot);
       }
-      if (organization != null) {
-        request = ModelMutations.update(organization);
+      if (newOrg != null) {
+        request = ModelMutations.update(newOrg);
         response = await Amplify.API.mutate(request: request).response;
         return "Success";
```
Comment (justend29) [inline lib/gateway.dart:363]: What happens if this request fails?

File: lib/gateway.dart [hunk 2]
```diff
@@ -367,17 +373,23 @@ class Gateway {
...
+            officers: organization?.officers,
+            parkingLots: const <ParkingLot>[]
+          );
         }
       }
-      if (organization != null) {
-        request = ModelMutations.update(organization);
+      if (replaced != null) {
+        request = ModelMutations.update(replaced);
         response = await Amplify.API.mutate(request: request).response;
```
Comment (justend29) [inline lib/gateway.dart:393]: same here, what if this fails?

File: lib/gateway.dart (hunk 1 above)
Comment (ChrisRuff) [reply lib/gateway.dart:363]: Its caught in the try catch below

File: lib/gateway.dart (hunk 2 above)
Comment (ChrisRuff) [reply lib/gateway.dart:393]: Its caught in the try catch below

File: amplify/backend/api/parkrGql/schema.graphql [hunk 3]
```diff
@@ -55,11 +52,6 @@ type GeoCoord {
     longitude: Float!
 }
```
[the remark this answers is not in the document]
Comment (ChrisRuff) [reply amplify/backend/api/parkrGql/schema.graphql:53]: Doesnt seem like it, codegen works

**Review by justend29 — APPROVED** [review]

**Pull request merged**